parse-opcodes

```diff
@@ -767,6 +796,28 @@ def make_chisel():
   print '  }'
   print '}'
 
+def print_go_insn(name):
+  print '\tcase A%s:' % name.upper().replace('.', '')
+  # Yank bits 0-6 of the encoded instruction (i.e., the opcode) and print them.
+  print '\t\treturn 0x%x' % yank(match[name], 0, 7)
```

Comment:
How are you tracking the rest of the bits in the instructions (i.e. the funct fields)? Maybe I just don't understand how the port will use this file.